config/routes.rb

```diff
@@ -9,7 +9,7 @@
   devise_scope :user do
     get "sign_in", to: "users/sessions#new", as: :new_session
     post "sign_in", to: "users/session#create", as: :session
-    get "sign_out", to: "users/sessions#destroy", as: :destroy_user_session
+    # get "sign_out", to: "users/sessions#destroy", as: :destroy_user_session
```

Comment:
Is the user no longer able to sign out?

Reply:
> Is the user no longer able to sign out?

@jrhoads this route setup was causing routing issues earlier. However, I've just checked (after you pointed this out) and the application has stopped raising the duplicate route error. I suspect my earlier attempt of using a much newer version of devise might have been causing this.

I've uncommented the route.

Thank you!